gen/definitions/vlan_tag.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: VLAN Tag
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/vlantags
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: true
+    example: vlan_tag_1
+  - model_name: description
+    type: String
+    description: Optional user-created description.
+    example: "My TAG id"
+    mandatory: true
+  - model_name: overridable
+    type: Bool
+    description: Indicates whether object values can be overridden.
+    exclude_example: true
+    test_value: "true"
```

Comment:
We normally just include an `example` and don't require a `test_value` to be defined. `test_value` is only required if we need to make use of value only available at runtime, ie. something that needs to be retrieved via a data source (in pre-requisites).

Reply:
There are two scenarios for VLAN_TAG creation:
1. forcing to set override to TRUE and skipping startTag and endTag
2. setting startTag and endTag and living override
both method tested,
finally i have changed according to your request, that case is more useful and natural